### PR TITLE
Feat: add mobile design

### PR DESCRIPTION
**Pull request by Yinkusadel**
### Here is the link to the [design](https://www.frontendmentor.io/challenges/calculator-app-9lteq5N29)

### CODE REVIEW

File: axe-check.js [hunk 1]
```diff
@@ -46,10 +46,9 @@ async function runAccessibilityCheck() {
...
       report += '</details>\n\n';
     }
-
   }
   fs.writeFileSync('accessibility_report.md', report, 'utf8');
   console.log('Accessibility report generated.');
 }
 
-runAccessibilityCheck();
+runAccessibilityCheck();
```
Comment (moshoodabdullahi) [inline axe-check.js]: You seem to be having an issue in your axe-check.js, I think is a no-next-line error or something, try giving it a new line after that line 54

**Review by moshoodabdullahi — CHANGES_REQUESTED** [review]
image
Take a look at this, you didn't add assignment link to your PR and also see to the changes required down there and then you are good to go

File: themetwo.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+              <div class="label-input">
+                <div class="labels">
+                  <label for="theme1">1</label>
+                  <label for="theme2">2</label>
+                  <label for="theme3">3</label>
+                </div>
+                <div class="theme-inputs">
+                  <input id="theme1" name="theme" type="radio" checked="" />
+                  <input id="theme2" name="theme" type="radio" />
+                  <input id="theme3" name="theme" type="radio" />
```
Comment (teckcipher) [inline themetwo.html]: theme two should be checked here

File: themethree.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                <div class="labels">
+                  <label for="theme1">1</label>
+                  <label for="theme2">2</label>
+                  <label for="theme3">3</label>
+                </div>
+                <div class="theme-inputs">
+                  <input id="theme1" name="theme" type="radio" checked="" />
+                  <input id="theme2" name="theme" type="radio" />
+                  <input id="theme3" name="theme" type="radio" />
+                </div>
```
Comment (teckcipher) [inline themethree.html]: theme three should be checked here

File: themethree.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    <link href="css/index.css" rel="stylesheet" />
+  </head>
+  <body class="theme-three">
+    <main>
+      <div class="pages-body">
+        <div class="calculator-container">
+          <div class="calculator-mainbody">
+            <div class="header">
+              <div class="header-texts">
+                <p class="calc-text">calc</p>
```
Comment (teckcipher) [inline themethree.html]: This can be the h1

**Review by teckcipher — CHANGES_REQUESTED** [review]
See https://github.com/moshoodabdullahi/js-calculator/pull/2#pullrequestreview-1708200746 for more reviews

Nice job 👏

**Review by moshoodabdullahi — DISMISSED** [review]
It all looks good to me

File: css/index.css [hunk 5]
```diff
@@ -1,9 +1,247 @@
...
+  --reset-key-background-hover: hsl(280deg 56% 44%);
+  --reset-key-box-shadow: hsl(285deg 91% 52%);
+  --equal-to-key-background: hsl(176deg 100% 44%);
+  --equal-to-key-background-hover: hsl(177deg 100% 79%);
+  --equal-to-key-box-shadow: hsl(177deg 92% 70%);
+  --white: hsl(0deg 0% 98%);
+}
+
+.pages-body {
+  width: 100%;
```
Comment (teckcipher) [inline css/index.css:76]: this can be width-full 🤷‍♂️

File: css/index.css [hunk 6]
```diff
@@ -1,9 +1,247 @@
...
+}
+
+.calculator-container {
+  width: 70%;
+  min-height: 100vh;
+}
+
+.calculator-mainbody {
+  min-height: 90vh;
+  width: 100%;
```
Comment (teckcipher) [inline css/index.css:89]: width-full 🤷‍♂️

File: css/index.css [hunk 7]
```diff
@@ -1,9 +1,247 @@
...
+}
+
+.header {
+  width: 100%;
```
Comment (teckcipher) [inline css/index.css:93]: width-full 🤷‍♂️

File: css/index.css [hunk 8]
```diff
@@ -1,9 +1,247 @@
...
+  width: 90%;
+  height: 70px;
+}
+
+.theme-inputs input {
+  width: 20px;
+  height: 20px;
+  border-radius: 50%;
+  transition: 0.2s;
+  cursor: pointer;
```
Comment (teckcipher) [inline css/index.css:129]: This should be a general style for all input and buttons

File: css/index.css [hunk 9]
```diff
@@ -1,9 +1,247 @@
...
+  background-color: var(--mobile-radio-input-checked);
+}
+
+input:checked:hover {
+  background-color: var(--mobile-radio-input-checked-hover);
+}
+
+.user-input-display {
+  background-color: var(--user-input-background);
+  width: 100%;
```
Comment (teckcipher) [inline css/index.css]: width-full 🤷‍♂️

File: css/index.css [hunk 10]
```diff
@@ -1,9 +1,247 @@
...
+
+.reset-btn {
+  width: 48%;
+  font-size: 4vw;
+  font-weight: 700;
+  line-height: 3.7rem;
+  background-color: var(--reset-key-background);
+  color: var(--white);
+  border-radius: 0.3rem;
+  border: none;
```
Comment (teckcipher) [inline css/index.css]: Same here, all button should be border none by default - in the reset.css

File: css/index.css [hunk 11]
```diff
@@ -1,9 +1,247 @@
...
+
+.equal-to-btn {
+  width: 48%;
+  font-size: 4vw;
+  font-weight: 700;
+  line-height: 3.7rem;
+  background-color: var(--equal-to-key-background);
+  color: var(--white);
+  border-radius: 0.3rem;
+  border: none;
```
Comment (teckcipher) [inline css/index.css]: Same here

File: css/utility.css [hunk 12]
```diff
@@ -9,3 +9,19 @@
 .d-block {
   display: block;
 }
+
+.appearance {
+  appearance: none;
+}
```
Comment (teckcipher) [inline css/utility.css:15]: Use display none or visibility hidden instead, this style is not as stable as the mentioned ones esp for older browsers.

File: index.html [hunk 13]
```diff
@@ -6,9 +6,85 @@
...
+              <div class="user-input d-flex align-center">399,981</div>
+            </div>
+
+            <div class="user-butns">
+              <div class="calculator-btns">
+                <div class="row d-flex justify-space-between align-center">
+                  <button class="btn" type="button" value="7">7</button>
+                  <button class="btn" type="button" value="8">8</button>
+                  <button class="btn" type="button" value="9">9</button>
+                  <button class="delete-btn" type="button">DEL</button>
```
Comment (teckcipher) [inline index.html]: value attribute?

**Review by teckcipher — CHANGES_REQUESTED** [review]

**Review by teckcipher — DISMISSED** [review]

**Review by moshoodabdullahi — DISMISSED** [review]

File: css/utility.css (hunk 12 above)
Comment (Yinkusadel) [reply css/utility.css:15]: I've tried that ,they make the whole input disappear which isn't what i'm trying to achieve , also i tried using the before and after pseudo class to manipulate it which didn't work eventually ...

**Review by teckcipher — DISMISSED** [review]

**Review by moshoodabdullahi — APPROVED** [review]

**Review by teckcipher — APPROVED** [review]

**Pull request merged**